examples/discovery-search-app/src/App.js

```diff
@@ -85,9 +88,20 @@ function SearchPage() {
   );
 }
 
-function PreviewPage() {
+function PreviewPage({ document }) {
   const { setSelectedResult } = useContext(SearchApi);
 
+  const tabs = [
+    {
+      name: 'Document',
+      Component: DocumentPreview
+    }
+  ];
+
+  if (canRenderCIDocument(document)) {
+    tabs.push({ name: 'CI', Component: CIDocument });
```

Comment:
I know it's long, but I think it would be better to have this as `Content Intelligence` instead of `CI`, which may not be easily recognizable by users.